jetty-server/src/main/java/org/eclipse/jetty/server/AsyncContentProducer.java

```diff
@@ -166,7 +166,7 @@ public boolean consumeAll()
         if (LOG.isDebugEnabled())
         {
             x = new IOException("Unconsumed content");
-            LOG.debug("consumeAll {}", this);
+            LOG.debug("consumeAll [e={}] {}", x, this);
```

Comment:
This would be better as ...

``` java
LOG.debug("consumeAll {}", this, x);
```

Reply:
Do you think logging this exception's stack trace here would be useful?

Reply:
Yes, that is the whole point of creating a fresh new `IOException`, to capture the stack trace so that we know from where the call comes from.
If we don't print the stack trace, then there is no point creating a fresh new `IOException`.